packages/backend-proxy-middleware/README.md

```diff
@@ -0,0 +1,149 @@
+#  `@sap-ux/backend-proxy-middleware`
+
+The `@sap-ux/backend-proxy-middleware` is a [Custom UI5 Server Middleware](https://sap.github.io/ui5-tooling/pages/extensibility/CustomServerMiddleware) for proxying requests to backend servers. The middleware is agnostic to running in local environments and SAP Business Application Studio. Additionally, if credentials are maintained with the SAP Fiori tools then these will be used.
+
+It can be used either with the `ui5 serve` or the `fiori run` commands.
+
+## Configuration Options
+
+### `backend`
+| Option        | Type | Description |
+| ------------- | ------------- | ----------- |
+| `url`         | `string` mandatory (local)  | Mandatory URL pointing to the backend system. *Not required if destination is provided and the proxy is running SAP Business Application Studio |
+| `destination` | `string` mandatory (if no url) | Required if the backend system is available as destination in SAP Business Application Studio. |
+| `destinationInstance` | `string` optional | If a destination needs to be read by a specific instance of a destination service then you need to provide the id of the service as optional property `destinationInstance`.|
+| `path`        | `string` mandatory     | Path that is to be proxied |
+| `pathPrefix`  | `string` optional      | If provided then the path will be replaced with the pathPrefix before forwarding |
+| `client`      | `string|number` optional      | sap-client parameter |
```

Comment:
To be on the safe side, `client` needs to be provided as a `string`. Otherwise any number that starts with `0` will be interpreted as hexadecimal by the yaml parser. Hence `010` will be parsed to `10`.